src/Microsoft.AspNetCore.All/Microsoft.AspNetCore.All.csproj

```diff
@@ -11,6 +11,7 @@
 
   <ItemGroup>
     <!-- set PrivateAssets=None to ensure that all assets, including Build and Analyzer, are included in the nuspec -->
+    <PackageReference Include="Microsoft.AspNetCore" Version="$(AspNetCoreVersion)" PrivateAssets="None" />
```

Comment:
Not sure, guessing it was a judgement based on it having no assets of its own previously.